cdap-api/src/main/java/co/cask/cdap/api/schedule/ScheduleBuilder.java

```diff
@@ -86,7 +88,7 @@
    * JVM's default time zone will be used to interpret {@code startTime} and {@code endTime}.
    *
    * @param startTime the start time (inclusive) in the format of "HH:mm", for instance 2am should be "02:00"
-   * @param endTime the end time (exclusive) in the format of "HH:mm", for instance 5pm should be "17:00"
+   * @param endTime   the end time (exclusive) in the format of "HH:mm", for instance 5pm should be "17:00"
```

Comment:
Why add empty space? We don't align the parameter docs, as it is unnecessary and won't affect the output javadoc.